tesseract/tesseract_planning/tesseract_process_managers/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ add_library(${PROJECT_NAME} SHARED
     src/process_generators/motion_planner_process_generator.cpp
     src/process_managers/freespace_process_manager.cpp
     src/process_managers/raster_process_manager.cpp
-    src/taskflow_generators/sequential_failure_tree_taskflow.cpp)
+    src/taskflow_generators/sequential_taskflow.cpp)
 target_link_libraries(${PROJECT_NAME} PUBLIC
```

Comment:
Should this be linking tesseract::tesseract_command_language?

Reply:
Yes. It is in on my machine but may have added it over the weekend. I will push my changes soon.

Reply:
pushed my changes